content/api/motor-board.md

```diff
@@ -16,22 +16,22 @@ When you have more than one Motor Board connected to your kit, they can be acces
 motor_board = r.motor_boards['SERIAL']
 ```
 
-This board object has a property for each of the motors connected to it. These are mapped to the `m1` and `m0` variables. The boards are labelled so you know which motor is which.
+This board object has a property for each of the motors connected to it. These are mapped to the `m0` and `m1` variables. The boards are labelled so you know which motor is which.
 
 ```python
 r.motor_board.m0
 r.motor_board.m1
 ```
 
-## Powering Motors
-Motor power is controlled using PWM with 100% power being a duty cycle of 1. You set the power with a value between `-1` and `1` inclusive (where a negative value puts the motor in reverse).
+## Powering motors
+Motor power is controlled using [pulse-width modulation (PWM)][pwm]. You set the power with a fractional value between `-1` and `1` inclusive, where `1` is maximum speed in one direction, `-1` is maximum speed in the other direction and `0` sets the speed to zero.
```

Comment:
To match with the rest of the docs, can we keep the link href inline in the markdown

Reply:
> `0` sets the speed to zero

This isnt necessarily correct, or the wording should be changed at least. If the power is set to `0`, then the motors start braking, to me _setting the speed to `0`_ is closer to coasting?

Reply:
Hmm - personally I interpret "setting the speed to 0" to mean "the motor will try its best to make its speed 0", consistent with how it behaves when you set the speed to +1, -1, or any other value.

Either way, I'll clarify this by explicitly saying "braking" instead.